1001-2000/LIT1824Mafteh.xml

```diff
@@ -77,6 +78,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <relation name="ecrm:CLP46i_may_form_part_of" active="LIT5139MagicPrayer" passive="LIT1824Mafteh"></relation>
 
          </listRelation>
+         <div type="edition">
+            <note>The following text part description is based on the manuscript <ref type="mss" corresp="MEAG002"/>which is incomplete.</note>
```

Comment:
I would not add edition here at all, use it just in the manuscript for now. Edition for text must be from an edited text or at least from a best witness. 




VERY VERY IMPORTANT: never ever use diacritics or any non-ASCII symbols in any file names, folder names, branch names, or xml:IDs. The name of this branch is a problem.